huaweicloud/services/apig/resource_huaweicloud_apig_vpc_channel.go

```diff
@@ -1,293 +1,356 @@
 			"protocol": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Default:  PROTOCOL_TCP,
+				Default:  string(ProtocolTypeTcp),
 				ValidateFunc: validation.StringInSlice([]string{
-					PROTOCOL_TCP, PROTOCOL_HTTP, PROTOCOL_HTTPS,
+					string(ProtocolTypeTcp),
+					string(ProtocolTypeHttp),
+					string(ProtocolTypeHttps),
 				}, false),
+				Description: "The rotocol for performing health checks on backend servers in the VPC channel.",
 			},
 			"path": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The destination path for health checks.",
 			},
 			"healthy_threshold": {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      2,
 				ValidateFunc: validation.IntBetween(2, 10),
+				Description: "The the healthy threshold, which refers to the number of consecutive successful " +
+					"checks required for a backend server to be considered healthy.",
 			},
 			"unhealthy_threshold": {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      5,
 				ValidateFunc: validation.IntBetween(2, 10),
+				Description: "The unhealthy threshold, which refers to the number of consecutive failed checks " +
+					"required for a backend server to be considered unhealthy.",
 			},
 			"timeout": {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      5,
 				ValidateFunc: validation.IntBetween(2, 30),
+				Description:  "The timeout for determining whether a health check fails, in second.",
 			},
 			"interval": {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      10,
 				ValidateFunc: validation.IntBetween(5, 300),
+				Description:  "The interval between consecutive checks, in second.",
 			},
 			"http_code": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The response codes for determining a successful HTTP response.",
 			},
-			"members": {
-				Type:     schema.TypeSet,
-				Required: true,
-				MaxItems: 10,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"ip_address": {
-							Type:     schema.TypeString,
-							Optional: true,
-						},
-						"id": {
-							Type:     schema.TypeString,
-							Optional: true,
-						},
-						"weight": {
-							Type:         schema.TypeInt,
-							Optional:     true,
-							Default:      1,
-							ValidateFunc: validation.IntBetween(1, 100),
-						},
-					},
-				},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time when the VPC channel was created.",
 			},
 			"create_time": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Deprecated:  "Use 'created_at' instead",
+				Description: `schema: Deprecated; The time when the environment was created.`,
 			},
```

Comment:
This is  just an attribute, how about just remove it

Reply:
done.